galaxy_importer/loaders.py

```diff
@@ -281,7 +283,29 @@ def _get_readme(self):
 
     def _get_metadata_description(self):
         self.log.info('Getting role description')
-        return ''
+        description = None
+        metadata_file = self._get_metadata_file(self.rel_path)
+
+        if not metadata_file:
+            self.log.warning('No role metadata found')
+            return description
+
+        with open(metadata_file) as fp:
+            role_metadata = yaml.safe_load(fp)
+        try:
+            description = role_metadata['galaxy_info']['description']
+        except KeyError:
+            self.log.warning('No role description found in role metadata')
+        return description
+
+    @staticmethod
+    def _get_metadata_file(rel_path):
+        """Gets path to role metadata file."""
```

Comment:
(this is ok as is, just comment inspired by it)
I might lean towards calling this 'find_metadata_files' or similar, since it is searching multiple candidates and may return nothing. But that's just naming stuff, and 'find_*' is just a personal idiom I sometimes use.

I might also call it 'find_metadata_file_path' or 'get_metadata_file_path' but again just a vague suggestion. I am very inconsistent in how I name vars for things like file objects, file paths, full file paths, directory names, etc. But sometimes I find it helpful to be explicit about if something is a file (object), filename, or full path to a file.

Reply:
good suggestion, explicit is better here